### PR TITLE
H7: fix GMLAN bitbang timer

### PR DESCRIPTION
**Pull request by briskspirit**

### CODE REVIEW

File: tests/hitl/3_usb_to_can.py [hunk 1]
```diff
@@ -119,6 +119,14 @@ def test_gmlan_bad_toggle(p):
     assert comp_kbps_normal > (0.6 * SPEED_NORMAL)
     assert comp_kbps_normal < (1.0 * SPEED_NORMAL)
 
+@pytest.mark.panda_expect_can_error
+@pytest.mark.skip_panda_types(PandaGroup.GMLAN)
+def test_gmlan_bitbang(p):
```
Comment (adeebshihadeh) [inline tests/hitl/3_usb_to_can.py:124]: does this test catch the bug? it was pretty subtle

File: tests/hitl/3_usb_to_can.py [hunk 2]
```diff
@@ -119,6 +119,14 @@ def test_gmlan_bad_toggle(p):
...
+  p.set_safety_mode(Panda.SAFETY_ALLOUTPUT)
+  for _ in range(10):
+    p.can_send(0x10, b"data", 3)
+    time.sleep(0.1)
+  assert p.health()['gmlan_send_errs'] == 0
```
Comment (adeebshihadeh) [inline tests/hitl/3_usb_to_can.py:129]: also check the messages were sent on the loopback bus? do they get added there?

File: tests/hitl/3_usb_to_can.py (hunk 1 above)
Comment (adeebshihadeh) [inline tests/hitl/3_usb_to_can.py:124]: ideally, this test runs together with a white panda so we can read the message back

Comment (briskspirit) [reply tests/hitl/3_usb_to_can.py:124]: bug was caused by wrong timer settings, so yes, was caught on RP everytime

File: tests/hitl/3_usb_to_can.py (hunk 2 above)
Comment (briskspirit) [reply tests/hitl/3_usb_to_can.py:129]: don't think we ever forwarded those back, only rejected messages by safety mode.

File: tests/hitl/3_usb_to_can.py (hunk 1 above)
Comment (briskspirit) [reply tests/hitl/3_usb_to_can.py:124]: discussed

**Comment by adeebshihadeh** [comment]
Can we finish this?

**Pull request merged**